src/backend/services/users-api/tests/functional/test_routes.py

```diff
@@ -141,19 +144,19 @@ def test_get_all_users(test_client, test_db_patch):
 
     response = test_client.get(url)
 
-    assert b'"umnetID":"SHARMAA2"' in response.data
-    assert b'"umnetID":"SACHDEV1"' in response.data
+    assert b'"umnetId":"SHARMAA2"' in response.data
+    assert b'"umnetId":"SACHDEV1"' in response.data
 
 
-def test_create_error_duplicate_umnetID_and_password(test_client, test_db_patch, json_header):
+def test_create_error_duplicate_umnetId_and_password(test_client, test_db_patch, json_header):
 
     url = '/create'
 
-    # duplicate umnetID
+    # duplicate umnetId
     payload = {
         "first_name": "Zack",
         "last_name": "Holmberg",
-        "umnetID": "SHARMAA2",
+        "umnetId": "SHARMAA2",
         "password": "zack123",
         "public_key": "zack_pk"
```

Comment:
Can remove all public_key fields in tests

Reply:
we still need to store public keys in the user database for retrieval later.